### PR TITLE
updated files for Phase 1 delivery of refactored instruqt tracks

### PR DESCRIPTION
**Pull request by reselbob**
These tracks have been refactored to follow the new file format supported by Instruqt. Also, I copyedited and formatted the content in the tracks I updated. 

I made changes in the following  tracks.

- Getting started: https://github.com/openshift-instruqt/instruqt/tree/master/developing-on-openshift/getting-started
- Logging in to an OpenShift Cluster: https://github.com/openshift-instruqt/instruqt/tree/master/using-the-cluster/cluster-access
- Build and deploy applications with odo: https://github.com/openshift-instruqt/instruqt/tree/master/developing-on-openshift/developing-with-odo
- Deploy applications from images: Deploy applications from images
- Deploy applications from source code: https://github.com/openshift-instruqt/instruqt/tree/master/using-the-cluster/deploying-python
- Manage resource objects: https://github.com/openshift-instruqt/instruqt/tree/master/using-the-cluster/resource-objects
- Access and expose a database: https://github.com/openshift-instruqt/instruqt/tree/master/using-the-cluster/port-forwarding
- Transfer files in and out of containers: https://github.com/openshift-instruqt/instruqt/tree/master/using-the-cluster/transferring-files
- Getting started with Quarkus: https://github.com/openshift-instruqt/instruqt/tree/master/developing-with-quarkus/getting-started
- Quarkus for Spring Boot developers: https://github.com/openshift-instruqt/instruqt/tree/master/developing-with-quarkus/spring

Once the pull request is reviewed, accepted and merged, then the command to update the track using the `instruqt` CLI is to go the track directory and execute the following command: `instruqt track push`

For example:
```
cd using-the-cluster/port-forwarding/
instruqt track push
```

### CODE REVIEW

**Review by danieloh30 — APPROVED** [review]
Quarkus Getting-Started and Spring tracks look good to me.

File: developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+  type: terminal
+  hostname: crc
+difficulty: intermediate
+timelimit: 360
+---
+In this topic you will learn how to transfer files to and from a container. You'll create an application from an existing container image hosted in the Quay.io container registry. Then you will access a command line shell within the application's underlying Linux container. Finally, you'll use the `oc rsync` command from the shell's command line to copy a file out of the container and into the local machine.
+
+|NOTE|
+|----|
+|This topic uses the `oc rsync` command many time. Thus, you might want to learn the details of the command before proceeding. Click this [link](https://docs.openshift.com/container-platform/3.11/dev_guide/copy_files_to_container.html) to go to a in-depth discussion of  `oc rsync`|
```
Comment (cedricclyburn) [inline developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md]: Small typo for "command many time"

File: developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+----
+
+`Step 9:`  Run the following to exit the interactive shell and return to the local machine:
+
+```
+exit
+```
+
+Next you will use the `oc rsync` command to copy files from the container into the local machine.
```
Comment (cedricclyburn) [inline developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md]: Perhaps mention that there's no need to run this command in this step just yet? I got confused and ran it lol

**Comment by cedricclyburn** [comment]
Howdy! I wasn't able to access your version of `getting started`, `odo`, and  `deploying from source code`, but I reviewed the rest of them and they look good to me! Great work on all of this, I'm assuming the others are good to go as well. Just some general info about how long it took me as an average developer to walk through these:
- Logging in to an OpenShift Cluster: 10 minutes after environment setup
- Deploy applications from images: 15 minutes after environment setup
- Manage resource objects: 25 minutes after environment setup
- Access and expose a database: 20 minutes after environment setup
- Transfer files in and out of containers: 20 minutes after environment setup

**Review by cedricclyburn — APPROVED** [review]

File: developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md (hunk 2 above)
Comment (reselbob) [reply developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md]: Changed the language to this: In upcoming steps you will use the `oc rsync` command to copy files from the container into the local machine.

File: developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md (hunk 1 above)
Comment (reselbob) [reply developing-on-openshift/transferring-files/02-02-downloading-files-from-a-container/assignment.md]: fixed typo

**Pull request merged**